tools/cellxgene_census_builder/src/cellxgene_census_builder/build_state.py

```diff
@@ -20,38 +20,70 @@
 
 CENSUS_BUILD_CONFIG = "config.yaml"
 CENSUS_BUILD_STATE = "state.yaml"
-CENSUS_CONFIG_DEFAULTS = {
-    # General config
-    "verbose": 1,
-    "log_dir": "logs",
-    "log_file": "build.log",
-    "reports_dir": "reports",
-    "consolidate": True,
-    "disable_dirty_git_check": True,
-    "dryrun": False,  # if True, will disable copy of data/logs/reports/release.json to S3 buckets. Will NOT disable local build, etc.
+
+
+@define
```

Comment:
what is our coding convention around fully naming short / ambiguous names like this?  I personally prefer @attrs.define, but I will happily follow group conventions.

Reply:
The attrs doc used this convention, so I adopted it, but I am not very opinionated.

Reply:
most of the Builder uses the other convention. Given the diversity out there, I feel we should have a convention in our code base.

Happy to do whatever is best, but one convention seems wise.  I didn't see a good default from Google Py conventions.   And I don't see any reason to hold up this PR for the discussion - just wanted to raise the change as notable given the rest of the code base.

Reply:
We can revert to the previous convention, I am just a bit worried about the verbosity.

Reply:
OK, I can live with the change - this is not a major deal :-)